ports/nrf/drivers/bluetooth/ble_drv.c

```diff
@@ -130,14 +135,22 @@ uint32_t ble_drv_stack_enable(void) {
         .source = NRF_CLOCK_LF_SRC_RC,
         .rc_ctiv = 16,
         .rc_temp_ctiv = 2,
+#if (BLUETOOTH_SD == 132 && BLE_API_VERSION == 4)
+        .accuracy = NRF_CLOCK_LF_ACCURACY_250_PPM
```

Comment:
this might differ between boards - should it be defined in the board configuration?

Reply:
My thinking is to keep some sane default for the PPM at this point. The boards supported in this port is OK with using fixed values. I'm inclined to postpone making this configurable until there there is a use case which provokes this to be confgurable. Until then, is it reasonable to keep static values?

Reply:
That should be fine. When some boards begin to misbehave (dropped connections etc.) it might need to be changed.